understanding/dragging-movements.html

```diff
@@ -56,213 +56,154 @@
+               <p>この達成基準の目的は、ドラッグ動作を使用する機能が、要素のドラッグに必要な器用さを必要とせずに、別の<a href="#dfn-single-pointer">シングルポインタ</a>操作モードを持つことを保証することである。
                </p>
-               <p>Some people cannot perform dragging movements in a precise manner. Others use a specialized
-                  or adapted input device, such as a trackball, head pointer, eye-gaze system, or speech-controlled
-                  mouse emulator, which may make dragging cumbersome and error-prone.
+               <p>ドラッグ動作を正確に実行できない人もいる。また、トラックボール、ヘッドポインタ、視線システム、音声制御マウスエミュレータなどの特殊な又は適応した入力デバイスを使用する場合もあるが、これによりドラッグが煩雑になり、エラーが発生しやすくなる場合がある。
                </p>
-               <p>When an interface implements functionality that uses dragging movements, users perform
-                  four discrete actions:
+               <p>インタフェースがドラッグ動作を使用する機能を実装する場合、利用者は次の四つの個別のアクションを実行する:
                </p>
                <ol>
                   
-                  <li>tap or click to establish a starting point, then</li>
+                  <li>タップ又はクリックして始点を確立し、</li>
                   
-                  <li>press and hold that contact while...</li>
+                  <li>その接点を押しながら、</li>
                   
-                  <li>performing a repositioning of the pointer, before...</li>
+                  <li>ポインタの位置変更を実行する前に、</li>
                   
-                  <li>releasing the pointer at the end point.</li>
+                  <li>終点でポインタを放す。</li>
                   
                </ol>
-               <p>Not all users can accurately press and hold that contact while also repositioning
-                  the pointer. An alternative method must be provided so that users with mobility impairments
-                  who use a pointer (mouse, pen, or touch contact) can use the functionality.
+               <p>全ての利用者が、その接点を正確に押し続けながら、ポインタの位置を変更できるとは限らない。ポインタ (マウス、ペン、またはタッチ接触) を使用する運動障害のある利用者が機能を使用できるように、代替方法を提供しなければならない。
                </p>
-               <p>This requirement is separate from keyboard accessibility because people using a touch
-                  screen device may not use a physical keyboard. Keyboard specific interactions such
-                  as tabbing or arrow keys may not be possible when encountering a drag and drop control.
-                  Note, however, that providing a text input can be an acceptable single-pointer alternative
-                  to dragging. For example, an input beside a slider could allow any user to enter a
-                  precise value for the slider. In such a situation, the on-screen keyboard that appears
-                  for touch users offers a single-pointer means of entering an alphanumeric value.
+               <p>タッチスクリーンデバイスを使用する人々は物理キーボードを使用しない可能性があるため、この要件はキーボードのアクセシビリティとは異なるものである。ドラッグ＆ドロップコントロールに遭遇したときに、タブキー、矢印キーなどのキーボード固有の操作ができない場合がある。ただし、テキスト入力を提供することは、ドラッグの代わりにシングルポインタとして受け入れられる可能性があることに注意する。例えば、スライダーの隣の入力は、利用者がスライダーに正確な値の記入を可能にする。そのような状況では、タッチの利用者に表示されるオンスクリーンキーボードは、シングルポインタで英数字を記入する手段を提供する。
```

Comment:
separate fromを単に「異なる」とするのは語弊がありそうです。違うか同じかを問題にしているわけではなく、別々に存在する (キーボードの達成基準とこの達成基準はそれぞれ満たす必要がある) という話ですね。分離されている、別個になっている、というくらいでしょうか。

例えば…以降はちょっと誤訳気味のよう気がしています。最後の文は "the on-screen keyboard ... offers a single-pointer" という文で、「オンスクリーンキーボードがシングルポインタを提供している」ということかと。多分こんな感じかなと思います。

```suggestion
               <p>タッチスクリーンデバイスを使用する人々は物理キーボードを使用しない可能性があるため、この要件はキーボードのアクセシビリティとは別個になっている。ドラッグ＆ドロップコントロールに遭遇したときに、タブキー、矢印キーなどのキーボード固有の操作ができない場合がある。ただし、テキスト入力を提供することは、ドラッグの代わりにシングルポインタとして受け入れられる可能性があることに注意する。例えば、スライダーの隣に入力欄があり、スライダーで設定したい値を利用者が正確に入力できる場合がある。そのような状況では、タッチの利用者に表示されるオンスクリーンキーボードが、英数字の値を入力するという手段によって、シングルポインタを提供していることになる。
```